forms.py

```diff
@@ -0,0 +1,10 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, PasswordField
+from wtforms.validators import Email, DataRequired, Length
+
+
+class RegistrationForm(FlaskForm):
+    name = StringField('Name', validators=[DataRequired()])
+    surname = StringField('Surname', validators=[DataRequired(), Length(min=6)])
+    email = StringField('Email', validators=[DataRequired(), Email()])

```

Comment:
Тут можно использовать `EmailField`
```suggestion
    email = EmailField('Email', validators=[DataRequired(), Email()])

```